packages/LHEXT.yaml

```
@@ -0,0 +1,28 @@
+---
+name: 'LHEXT'
+version: '1.34'
+release: 1
+summary: 'LZH decompression tool'
+author: 'Tomoya Iwata'
+package_author: 'Fubukimaru'
+license: 'Proprietary'
+category: 'Tools'
```

Being a file decompression tool, maybe the `Files` section would be more correct?

```suggestion
category: 'Files'
```